lib/BitrateTest.ts

```diff
@@ -175,13 +186,14 @@ export class BitrateTest extends EventEmitter {
     clearInterval(this._sendDataIntervalId!);
     clearInterval(this._checkBitrateIntervalId!);
 
-    this._pcSender.close();
-    this._pcReceiver.close();
+    this._getReport().then((report: BitrateTest.Report) => {
```

Comment:
If `this._getReport()` rejects does that mean we might not emit a report?

Reply:
It may also mean we don't clean up our peerconnections

Reply:
Good catch. I need to handle promise rejection here.